tracker/telegram/bot.py

```diff
@@ -164,6 +164,26 @@ async def send_available_issues(msg: Message) -> None:
 
         await msg.reply(message, parse_mode="HTML")
 
+@dp.message(F.text.contains("/issues "))
+async def get_contributor_tasks(message: Message):
+    _ , username = message.text.split(" ", 1)
+
+    regex = r"ODHack"
+
+    issues = get_user_issues(username, True, True, regex)
+
+    msg = "ODHack Issues assigned: \n"
+
+    if len(issues) > 0:
+        for issue in issues:
+            msg += (
+                    f"{issue}\n"
```

Comment:
Pls move this to `templates.py` (Probably you will need no sync with main one more time)